sdk/keyvault/azure-security-keyvault-secrets/CMakeLists.txt

```diff
@@ -46,10 +52,11 @@ target_include_directories(
   azure-security-keyvault-secrets
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
+      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
```

Comment:
Are you doing this so you can include the private headers as `include "private/header.hpp"` ??  
I think for private headers it is better if we include the directory as `include "../private/header.hpp"`

Reply:
yea, i do it for private headers, ok, i'll remove it